app/uk/gov/hmrc/customs/declarations/information/services/declarationsInformationServices.scala

```diff
@@ -54,7 +50,7 @@ class DeclarationStatusService @Inject()(statusResponseFilterService: StatusResp
       case "cds60001" => backendCDS60001NotFoundResponse
       case "cds60002" => backendCDS60002SearchInvalidResponse
       case "cds60003" => backendCDS60003InternalServerErrorResponse
-      case _          => ErrorInternalServerError
+      case _ => ErrorInternalServerError
```

Comment:
can these have the same indention as before?

Reply:
Not sure how this happened, but will do.